template/Android Studio 2/recipe.xml.ftl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<recipe>
+
+    <dependency mavenUrl="com.github.paolorotolo:appintro:4.1.0" />
```

Comment:
This may need to be edited as we are now using JitPack.io for future releases:
https://github.com/apl-devs/AppIntro/releases/tag/v4.2.0